website/docs/faqs/Git/github-permissions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "I'm seeing a 'GitHub and dbt Cloud latest permissions' error"
+description: "GitHub and dbt Cloud permissions error"
+sidebar_label: "GitHub and dbt Cloud permissions error"
+---
+
+If you're seeing a `This account needs to accept the latest permissions for the dbt Cloud GitHub App` error in the dbt Cloud IDE &mdash; this usually occurs when the permissions for the dbt Cloud GitHub App are out of date.
+
+To solve this issue, you'll need to update the permissions for the dbt Cloud GitHub App in your GitHub account. Here's a couple of ways you can do it:
+
+#### First workaround
+
+You'll need to reach out to your Github organization admin to update the permissions for the dbt Cloud GitHub App.
+
+1. Go directly to GitHub to determine if any updated permissions are required.
```

Comment:
What happens if they're not and they're still getting the error? Do they skip to the second workaround or something else?